### DISCUSSION TITLE
Variable Rate Injection Well

### DISCUSSION THREAD

**Discussion by CharlesHostetler** [GEOS-DEV/GEOS#2753] [Q&A]
I want to model a supercritical CO2 injection well whose target injection rate is a function of time.  The simplest example would be a 50 year total simulation time with injection at a constant rate for the first 20 years, then no injection after that to watch the pressure fall off after injection.  

I think I need to add this to my WellControls element:

targetTotalRateTableName="targetTotal RateTable"

and then create a text tile called targetTotalRateTable.txt

1. Is this the right approach?
2. What are the contents and formats of this file?

Thanks!

Charles Hostetler
SCS Engineers
chostetler@scsengineers.com

**Comment by paveltomin** [comment]
It is the right approach. Please check the cases (xml files) from repo to see examples how to use it:
https://github.com/search?q=repo%3AGEOS-DEV%2FGEOS%20targetTotalRateTableName&type=code
